src/hooks/useAuthModal.tsx

```diff
@@ -13,17 +12,28 @@ type ModalProps = {
 
 const AuthModal = ({ queryRef }: ModalProps) => {
   const { hideModal } = useModal();
-  const isAuthenticated = useIsAuthenticated();
 
   const query = useFragment(
     graphql`
       fragment useAuthModalFragment on Query {
+        viewer {
+          ... on Viewer {
+            user {
+              id
+            }
+          }
+          # TODO: handle ErrNotAuthorized type
```

Comment:
is this a todo for different PR?

Reply:
yeahh there are a lot of error-handling-related TODOs for the app in general, so will tackle then!